package.json

```diff
@@ -14,6 +14,14 @@
     "frontend"
   ],
   "devDependencies": {
-    "@changesets/cli": "^2.29.5"
-  }
-}
\ No newline at end of file
+    "@chainlink/contracts": "^1.4.0",
+    "@changesets/cli": "^2.29.5",
+    "@openzeppelin/contracts": "^5.4.0",
+    "@tailwindcss/postcss": "^4.1.12",
+    "autoprefixer": "^10.4.21",
+    "content-hash": "^2.5.2",
+    "postcss": "^8.5.6",
+    "tailwindcss": "^4.1.12"
```

Comment:
why are you importing these dependencies top level?

The smart contracts should be installed using forge.

The other dependencies should be installed in the frontend package I believe?

Reply:
yeah, you need to `cd contracts` then follow the contracts README. https://github.com/stigmergic-org/simplepage/blob/main/contracts/README.md

If you want you can add a note about this in the general README :)